data/json/vehicle_parts.json

```diff
@@ -726,6 +726,22 @@
         "breaks_into": "ig_vp_frame"
     },{
         "type" : "vehicle_part",
+        "id" : "box_locking",
+        "name" : "locked box",
+        "symbol" : "o",
+        "color" : "brown",
+        "broken_symbol" : "#",
+        "broken_color" : "brown",
+        "damage_modifier" : 60,
+        "durability" : 100,
+        "size" : 275,
+        "item" : "frame",
+        "difficulty" : 2,
+        "location" : "center",
+        "flags" : ["CARGO", "CARGO_LOCKING", "TOOL_SCREWDRIVER", "BOARDABLE", "COVERED"],
+        "breaks_into": "ig_vp_frame"
+    },{
```

Comment:
Vehicle parts support "copy-from". Don't specify everything from scratch.
Check out `data/json/vehicleparts/battery.json` for an example.

Reply:
Good point. I will do this